kornia/geometry/liegroup/se3.py

```diff
@@ -36,17 +39,19 @@ def __init__(self, r: So3, t: Tensor) -> None:
         Example:
             >>> from kornia.geometry.quaternion import Quaternion
             >>> q = Quaternion.identity(batch_size=1)
-            >>> s = Se3(So3(q), torch.ones((1,3)))
+            >>> s = Se3(So3(q), torch.ones((1, 3)))
             >>> s.r
-            real: tensor([[1.]], grad_fn=<SliceBackward0>)
-            vec: tensor([[0., 0., 0.]], grad_fn=<SliceBackward0>)
+            Parameter containing:
+            tensor([[1., 0., 0., 0.]], requires_grad=True)
             >>> s.t
-            tensor([[1., 1., 1.]])
+            Parameter containing:
+            tensor([[1., 1., 1.]], requires_grad=True)
         """
+        super().__init__()
         KORNIA_CHECK_TYPE(r, So3)
         KORNIA_CHECK_SHAPE(t, ["B", "3"])
```

Comment:
Getting some issues when t is passed as non batched, guess its a bug in KORNIA_CHECK_SHAPE.
```
(Pdb) tt = torch.ones(5)
(Pdb) Se3(So3.identity(1), tt)
rotation: Parameter containing:
tensor([[1., 0., 0., 0.]], requires_grad=True)
translation: Parameter containing:
tensor([1., 1., 1., 1., 1.], requires_grad=True)

```

Reply:
Yes, this needs to be addressed separately, for now we need to carefully pass both so3 and translation vector as non batched